scripts/commands/archive-list.php

```diff
@@ -25,6 +25,6 @@
 }
 
 echo sprintf(
-        "Shopping list '%s' has been archived.",
-        $model->getList(),
+    "Shopping list '%s' has been archived.",
+    $model->getList(),
     ) . PHP_EOL;
```

Comment:
Formatting: Close! Line 30 needs to have no indent. You've got the indent correct for lines 28 and 29 now.